packages/client/modules/teamDashboard/components/TeamDashMain/TeamDrawer.tsx

```diff
@@ -0,0 +1,111 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React, {useRef} from 'react'
+import {useFragment} from 'react-relay'
+import LabelHeading from '~/components/LabelHeading/LabelHeading'
+import {TeamDrawer_viewer$key} from '~/__generated__/TeamDrawer_viewer.graphql'
+import {Breakpoint, DrawerTypes, RightSidebar} from '../../../../types/constEnums'
+import AgendaListAndInput from '../AgendaListAndInput/AgendaListAndInput'
+import ManageTeamList from '../ManageTeam/ManageTeamList'
+import CloseSidebar from '../CloseSidebar/CloseSidebar'
+import ResponsiveDashSidebar from '../../../../components/ResponsiveDashSidebar'
+import {PALETTE} from '../../../../styles/paletteV3'
+import ToggleAgendaListMutation from '../../../../mutations/ToggleAgendaListMutation'
+import ToggleManageTeamMutation from '../../../../mutations/ToggleManageTeamMutation'
+import useAtmosphere from '../../../../hooks/useAtmosphere'
+import useMutationProps from '../../../../hooks/useMutationProps'
+import useBreakpoint from '../../../../hooks/useBreakpoint'
+
+const SidebarHeader = styled('div')({
+  alignItems: 'center',
+  display: 'flex',
+  justifyContent: 'space-between',
+  padding: '16px 8px 16px 16px'
+})
+
+const SidebarContent = styled('div')<{isDesktop: boolean}>(({isDesktop}) => ({
+  backgroundColor: PALETTE.WHITE,
+  display: 'flex',
+  overflow: 'hidden',
+  padding: `0 0 ${isDesktop ? 58 : 0}px`,
+  height: '100vh',
+  flexDirection: 'column',
+  width: RightSidebar.WIDTH
+}))
+
+const StyledLabelHeading = styled(LabelHeading)({
+  fontSize: 14,
+  textTransform: 'none'
+})
+
+interface Props {
+  viewer: TeamDrawer_viewer$key
+}
+
+const TeamDrawer = (props: Props) => {
+  const data = useFragment(
+    graphql`
+      fragment TeamDrawer_viewer on User {
+        dashSearch
+        team(teamId: $teamId) {
+          id
+          ...AgendaListAndInput_team
+          ...ManageTeamList_team
+        }
+        teamMember(teamId: $teamId) {
+          hideAgenda
+          hideManageTeam
+          manageTeamMemberId
+        }
+      }
+    `,
+    props.viewer
+  )
+  const {dashSearch, team, teamMember} = data
+  const hideAgenda = teamMember?.hideAgenda
+  const hideManageTeam = teamMember?.hideManageTeam
+  const manageTeamMemberId = teamMember?.manageTeamMemberId
```

Comment:
`team` and `teamMember` should never be null as we're grabbing the `team` and `teamMember` that belong to the viewer. I could destructure it, use a bang, and save a couple of lines but I did see a bug recently where a user deleted their account, signed up with the same email, and then didn't belong to any teams initially. So, I'd prefer to play it safe here